routes/api.php

```diff
@@ -1,9 +1,13 @@
 <?php
 
 Route::group(['middleware' => ['forcedomain'], 'as' => 'api::'], function () {
+    /* Routes for smartXp dmx_values (excluded from web and therefore https) */
+    Route::group(['middleware' => ['api']], function () {
```

Comment:
Routes in routes/api.php automatically get the api middleware assigned, so this line can be removed.

Reply:
I had not seen that, it is fixed now